docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -493,6 +493,7 @@ Query Parameters:
 -   **follow** – 1/True/true or 0/False/false, return stream. Default false
 -   **stdout** – 1/True/true or 0/False/false, show stdout log. Default false
 -   **stderr** – 1/True/true or 0/False/false, show stderr log. Default false
+-   **since** – UNIX epoch time to filter logs created since a timestamp. Default: 0.
```

Comment:
UNIX timestamp (integer) to filter logs. Specifying a timestamp will only output log-entries since that timestamp. Default: 0 (unfiltered).


(The UNIX epoch time is 01-01-1970; a UNIX *timestamp* is the number of seconds since that epoch)